test/suites/test_crud.py

```diff
@@ -0,0 +1,584 @@
+import re
+import sys
+import time
+import unittest
+import tarantool
+from .lib.tarantool_server import TarantoolServer
+from tarantool.error import CrudModuleError
+
+
+def create_server():
+    srv = TarantoolServer()
+    srv.script = 'test/suites/crud_server.lua'
+    srv.start()
+
+    return srv
+
+
+class TestSuite_Crud(unittest.TestCase):
```

Comment:
To actually run a test, you need to specify your file here:

https://github.com/tarantool/tarantool-python/blob/70b0a3debe5ffa7f1c0b5c84d6fb9e28cbe26152/test/suites/__init__.py#L26-L32

And after you do it, CI run will fail cause you never install `crud` module in the pipeline. So you need to fix it too.

Reply:
Done. At the moment I have problem with implementing CI tests under windows. AFAIU, there is a problem with the [availability of methods](https://github.com/tarantool/tarantool-python/actions/runs/3628331837/jobs/6119215806#step:11:1957), the reason for which I cannot yet determine. The rest of the cases are working properly

Reply:
Seems it is complicated to configure vshard replicasets under WSL so let's skip this one.